src/utils/rdf.js

```diff
@@ -0,0 +1,83 @@
+import { Parser } from 'n3';
+import createClient from 'ipfs-http-client';
+import { fromJsonldjsNode } from './claimgraph';
+
+const ipfsDefaultConfig = 'http://localhost:5001';
+
+let ipfsClient;
+
+/**
+ * Writes a document to IPFS and returns its CID
+ * @param {string} document - RDF document to store on IPFS
+ * @param {object} connectionConfig - IPFS HTTP Client connection options
+ * @returns {Promise<any>}
+ */
+export async function writeToIPFS(document, connectionConfig = ipfsDefaultConfig) {
```

Comment:
Consider this code:

```js
try {
  await writeToIPFS('yeah', 'http://localhost:5005');
} catch {
  await writeToIPFS('yeah', 'http://localhost:5001');
}
```

Reply:
do you think that user will want to connect to multiple nodes in one program?

Reply:
you know, i dont think its wise actually to include `ipfs-http-client` module in SDK and providing these very very simple methods to write for example. i think dereference could be useful if we pass the client from the caller, rather than in the SDK